WalletWasabi.Fluent/Views/AddWallet/RecoverWalletView.axaml

```diff
@@ -25,10 +25,10 @@
                 <c:TagsBox x:Name="SeedTagsBox"
                            DockPanel.Dock="Top"
                            ItemCountLimit="12"
+                           SuggestionsAreCaseSensitive="True"
```

Comment:
I dont think user entry is case sensitive here... they can be lazy and type lower case.. or upper case... but pressing enter on the suggestion will correct it.

Reply:
yeah that should be false, forgot to set it haha